### PR TITLE
Exchange Certs Feature

### PR DESCRIPTION
**Pull request by dpaulson45**

### CODE REVIEW

File: HealthChecker.ps1 [hunk 1]
```diff
@@ -4664,6 +4666,118 @@ param(
...
+    {
+        $analyzedResults = Add-AnalyzedResultInformation -Name "Valid Auth Certificate found" -Details $true `
+        -DisplayGroupingKey $keySecuritySettings `
+        -DisplayCustomTabNumber 1 `
+        -DisplayWriteType "Green" `
+        -AnalyzedInformation $analyzedResults
+    }
+    else
+    {   
+        $analyzedResults = Add-AnalyzedResultInformation -Name "Valid Auth Certificate found" -Details $false `
```
Comment (lusassl-msft) [inline HealthChecker.ps1:4768]: Let's change this to `Valid Auth Certificate found on server` to make it more clear that we're validating if the current auth certificate is in local machines certificate store.

File: HealthChecker.ps1 [hunk 2]
```diff
@@ -4664,6 +4666,118 @@ param(
...
+            $analyzedResults = Add-AnalyzedResultInformation -Details $namespace `
+            -DisplayGroupingKey $keySecuritySettings `
+            -DisplayCustomTabNumber 3 `
+            -AnalyzedInformation $analyzedResults
+        }
+    }
+
+    if($exchangeInformation.ExchangeCertificates.IsCurrentAuthConfigCertificate.Contains($true))
+    {
+        $analyzedResults = Add-AnalyzedResultInformation -Name "Valid Auth Certificate found" -Details $true `
```
Comment (lusassl-msft) [inline HealthChecker.ps1:4760]: Same here: `Valid Auth Certificate found **on server**``

**Review by lusassl-msft — COMMENTED** [review]
@dpaulson45 please have a look at my comments.

File: HealthChecker.ps1 [hunk 3]
```diff
@@ -4664,6 +4666,118 @@ param(
         }
     }
 
+    foreach($certificate in $exchangeInformation.ExchangeCertificates)
+    {
+        if($certificate.LifetimeInDays -ge 60)
+        {
+            $displayColor = "Green"
+        }
+        elseif($certificateLifetimeInDays -ge 30)
```
[the remark this answers is not in the document]
Comment (dpaulson45) [reply HealthChecker.ps1:4675]: Done in commit 93e9e3084450bfb37bfadce763e065b625452d6f

File: HealthChecker.ps1 (hunk 1 above)
Comment (dpaulson45) [reply HealthChecker.ps1:4768]: Done in commit 93e9e3084450bfb37bfadce763e065b625452d6f

File: HealthChecker.ps1 (hunk 2 above)
Comment (dpaulson45) [reply HealthChecker.ps1:4760]: Done in commit 93e9e3084450bfb37bfadce763e065b625452d6f

**Pull request merged**